tests/core/math/test_aabb.h

```diff
@@ -186,24 +186,52 @@ TEST_CASE("[AABB] Intersection") {
 			!aabb_big.intersects_plane(Plane(Vector3(0, 1, 0), 200)),
 			"intersects_plane() should return the expected result.");
 
+	Vector3 result, normal;
```

Comment:
```suggestion
	Vector3 result;
	Vector3 normal;
```
Prefer one declaration per line